flexmeasures/ui/crud/assets.py

```diff
@@ -195,6 +195,23 @@ class AssetCrudUI(FlaskView):
 
     route_base = "/assets"
 
+    def get_breadcrumb_info(self, asset: GenericAsset):
```

Comment:
Missing return type. I also suggest returning a dict instead.

Reply:
That way you can pass it as `**breadcrumb_info` to `render_flexmeasures_template`.

Reply:
Good point!